src/hooks/pools/lbp/rpc/decorators.tsx

```diff
@@ -0,0 +1,5 @@
+import { getPoolAccount as getLpbPollAccount } from './getPoolAccount';
+
+export const lbp = {
+  getLpbPollAccount,
```

Comment:
This does not work for 2 reasons:

1. the typo for `pool`
2. the decorator needs to have the same name as in the RPC method, which is `getPoolAccount`